apps/www/_blog/2024-06-18-calcom-platform-starter-kit-nextjs-supabase.mdx

```diff
@@ -0,0 +1,201 @@
+
+Initially the application was built to be run on SQLite. However, once requirements grew to include file storage, the Cal.com team remembered their Supabase frenemies and luckily, thanks to Prisma and Supabase, switching things over to Postgres three days before launch was a breeze.
+
+## Prisma configuration for usage with Postgres on Supabase
+
+When working with Prisma, your application will connect directly to your Postgres databases hosted on Supabase. To handle connection management efficiently, especially when working with serverless applications like Next.js, Supabase provides a connection pooler called [supavisor]() to make sure your database scales efficiently with your traffic.
+
+The configuration is specified in the `schema.prisma` file where you provide the following connection strings:
+
+```ts schema.prisma
+datasource db {
+  provider  = "postgresql"
+  url       = env("POSTGRES_PRISMA_URL")
+  directUrl = env("POSTGRES_URL_NON_POOLING")
+  schemas   = ["prisma"] // see multi-schema support below
+}
+```
+
+This loads the relevant Supabase connections strings from your `.env` file
+
+```bash .env
+POSTGRES_PRISMA_URL="postgres://postgres.YOUR-PROJECT-REF:[YOUR-PASSWORD]@aws-0-[REGION].pooler.supabase.com:6543/postgres" # Transaction Mode
+POSTGRES_URL_NON_POOLING="postgres://postgres.YOUR-PROJECT-REF:[YOUR-PASSWORD]@aws-0-[REGION].pooler.supabase.com:5432/postgres"  # Session Mode
+```
+
+You can find the values in the [Database settings](https://supabase.com/dashboard/project/_/settings/database) of your Supabase Dashboard.
+
+For more details on using Prisma with Supabase, read the [officiale docs](https://supabase.com/partners/integrations/prisma).
+
+### Multischema support in Prisma
+
+In Supabase the `public` schema is exposed via the autogenerated [PostgREST]() API, which allows you to connect with your database from any environment that speaks HTTPS using the [Supabase client libraries]() like [supabase-js]() for example.
+
+When using Prisma, your application will connect directly to your database, therefore it's advisable to put your data on a separate schema that is not exposed via the API.
+
+We can do this by enabling `multischema` support in the `schema.prisma` file:
+
+```ts schema.prisma
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["multiSchema"]
+}
+
+model Account {
+  id                String  @id @default(cuid())
+  // ...
+
+  @@schema("prisma")
+}
+```
+
+## React Dropzone and Supabase Storage for profile image uploads
+
+Supabase storage is an S3 compatible cloud-based object store that allows you to store files securely. It is conveniently integrated with [Supabase Auth]() allowing you to easily limit access for uploads and downloads.
+
+However, this project is using [next-auth]() instead of Supabase Auth. Luckily though, Supabase Storage is supa felxible, allowing you to easily create signed upload URLs server-side to then upload assets from the client-side.
```

Comment:
```suggestion
However, this project is using [next-auth]() instead of Supabase Auth. Luckily though, Supabase Storage is supa flexible, allowing you to easily create signed upload URLs server-side to then upload assets from the client-side.
```